.github/workflows/joystream-node-docker-dev.yml

```
@@ -47,15 +47,36 @@ jobs:
           export IMAGE_EXISTS=$(docker manifest inspect ${{ env.REPOSITORY }}:${{ steps.compute_shasum.outputs.shasum }} > /dev/null ; echo $?)
           echo "::set-output name=image_exists::${IMAGE_EXISTS}"
 
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        # uses: docker/setup-buildx-action@dc7b9719a96d48369863986a06765841d7ea23f6 # v1.7
+        uses: docker/setup-buildx-action@v2
+        id: builder1
+        if: ${{ steps.compute_image_exists.outputs.image_exists == 1 }}
+
+      - name: Cache Docker layers
+        uses: actions/cache@v3
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-dev
+        if: ${{ steps.compute_image_exists.outputs.image_exists == 1 }}
+
       - name: Build and push
-        uses: docker/build-push-action@v3
+        uses: docker/build-push-action@v4
         with:
           context: .
           file: joystream-node.Dockerfile
+          # arm64 cross compiling takes longer than 6h, so we only build for amd64
+          # platforms: linux/amd64,linux/arm64
```

```suggestion
          # platforms: linux/amd64
```
Since you said "only amd64" above.

---

The line is already commented out, but left it for reference if we want to add it back.
It might be better to drop this whole workflow and improve `joystream-node-docker.yml` which deploys two worker machines on AWS (amd64 and arm64) to build natively for each platform arch have the ansible playbook build for all the runtime profiles.